tests/python_tests/test_vlm_api.py

```diff
@@ -0,0 +1,111 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import openvino_genai
+import pytest
+import gc
+import os
+import numpy as np
+from PIL import Image
+from multiprocessing import Process
+
+from openvino_genai import VLMPipeline
+from openvino import Tensor
+from common import get_greedy, get_image_by_link, get_beam_search, get_greedy, get_multinomial_all_parameters
+
+def get_ov_model(model_dir):
+    import sys
+    from pathlib import Path
+    #TODO: use optimum-intel
+
+    sys.path.append(str(Path(__file__).resolve().parents[2] / 'samples/cpp/visual_language_chat'))
+    import importlib
+    export_MiniCPM = importlib.import_module("export_MiniCPM-V-2_6", "export_MiniCPM")
+    convert_llm = getattr(export_MiniCPM, "convert_llm")
+    convert_vision_encoder = getattr(export_MiniCPM, "convert_vision_encoder")
+    snapshot_download = getattr(export_MiniCPM, "snapshot_download")
+    patch_model_code = getattr(export_MiniCPM, "patch_model_code")
```

Comment:
may be it makes sense to execute code under `if` below, if they are really required?

Reply:
Agree, done.